services/data-info/src/data_info/routes/domain/stats.clj

```diff
@@ -9,6 +9,11 @@
 (def DataItemIdParam (describe UUID "The UUID of this data item"))
 (def DataItemPathParam (describe NonBlankString "The IRODS paths to this data item"))
 
+(s/defschema StatQueryParams
+  (assoc StandardUserQueryParams
+         (s/optional-key :validation-behavior)
+         (describe PermissionEnum "What level of permissions on the queried files should be validated?")))
```

Comment:
This is the start of several changes which allow the stat-gatherer endpoint to have a configurable level of file validation: just readability (as it always has), writeability, or ownership. This enabled removing some of the validators which depended on an iRODS connection.